hcfb-fe/tslint.json

```diff
@@ -6,31 +6,16 @@
     "arrow-return-shorthand": true,
     "callable-types": true,
     "class-name": true,
-    "comment-format": [
-      true,
-      "check-space"
-    ],
-    "curly": true,
```

Comment:
These `tslint.json` changes are made to keep tslint around for _style_ concerns, but disable it for _formatting_, based on this article:

https://medium.com/@victormejia/setting-up-prettier-in-an-angular-cli-project-2f50c3b9a537